src/Extension/Traits/FluentObjectTrait.php

```diff
@@ -109,12 +115,48 @@ protected function updateFluentCMSFields(FieldList $fields)
             'Locale' => 'Locale'
         ];
 
+        // Augment Localisation tab with clickable locale links to allow easy navigation between model localisations
+        if ($owner->hasExtension(CMSEditLinkExtension::class)) {
+            $columns->setFieldCasting([
+                // Make sure our links are correctly rendered (valid HTML)
+                'Title' => 'HTMLVarchar->RAW',
+            ]);
+
+            $controller = Controller::has_curr() ? Controller::curr() : null;
+            $request = $controller?->getRequest();
+
+            // Pass getVars separately so we can process them later
+            $params = $request?->getVars() ?? [];
+
+            // This is to get URL only, getVars are not part of the URL
+            $url = $owner->CMSEditLink();
+            $url = Director::makeRelative($url);
+
+            $summaryColumns['Title'] = [
+                'title' => 'Title',
+                'callback' => function (Locale $object) use ($url, $params) {
+                    if (!$object->RecordLocale()) {
+                        return null;
+                    }
+
+                    $recordLocale = $object->RecordLocale();
+                    $locale = $recordLocale->getLocale();
+                    $params['l'] = $locale;
+                    $localeLink = Controller::join_links($url, '?' . http_build_query($params));
+                    $localeTitle = Convert::raw2xml($recordLocale->getTitle());
+                    $render = sprintf('<a href="%s" target="_top">%s</a>', $localeLink, $localeTitle);
```

Comment:
Maybe we could put this in a SS template? I tend not to like putting HTML directly in PHP when we have an entire templating engine for this purpose.

Not a big deal.

Reply:
I feel like a whole template for a single HTML tag is overkill - especially since that means rendering it many many times for any given gridfield.